src/Tizen.NUI.Scene3D/src/public/ModelMotion/MotionIndex/BlendShapeIndex.cs

```diff
@@ -50,42 +53,66 @@ namespace Tizen.NUI.Scene3D
     /// </code>
-    /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// </example>
+    /// <since_tizen> 11 </since_tizen>
     public class BlendShapeIndex : MotionIndex
     {
         /// <summary>
         /// Create an initialized blend shape index.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 11 </since_tizen>
         public BlendShapeIndex() : this(Interop.MotionIndex.BlendShapeIndexNew(), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
         /// <summary>
-        /// Create an initialized blend shape index with invalid node id, and input blend shape id.
+        /// Create an initialized blend shape index with invalid node ID, and input blend shape ID.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <param name="blendShapeId">Blend shape ID for this motion index</param>
+        /// <since_tizen> 11 </since_tizen>
         public BlendShapeIndex(PropertyKey blendShapeId) : this(Interop.MotionIndex.BlendShapeIndexNew(PropertyKey.getCPtr(blendShapeId)), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
         /// <summary>
-        /// Create an initialized blend shape index with input node id, and input blend shape id.
+        /// Create an initialized blend shape index with input node ID, and input blend shape ID.
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <param name="modelNodeId">Node ID for this motion index</param>
+        /// <param name="blendShapeId">Blend shape ID for this motion index</param>
+        /// <since_tizen> 11 </since_tizen>
         public BlendShapeIndex(PropertyKey modelNodeId, PropertyKey blendShapeId) : this(Interop.MotionIndex.BlendShapeIndexNew(PropertyKey.getCPtr(modelNodeId), PropertyKey.getCPtr(blendShapeId)), true)
         {
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Create an initialized blend shape index with invalid node ID, and input blend shape string ID.
+        /// </summary>
+        /// <param name="blendShapeName">Blend shape string ID for this motion index</param>
+        /// <since_tizen> 11 </since_tizen>
+        public BlendShapeIndex(string blendShapeName) : this(Interop.MotionIndex.BlendShapeIndexNew(PropertyKey.getCPtr(new PropertyKey(blendShapeName))), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Create an initialized blend shape index with input node ID, and input blend shape ID.
```

Comment:
please replace "input " with "given " here and in lines 80, 91,   or just "with node ID and blend shape ID"

Reply:
Fix done.